WooCommerce/Classes/ViewRelated/AppCoordinator.swift

```diff
@@ -126,6 +130,45 @@ private extension AppCoordinator {
     }
 }
 
+// MARK: Store switching after store creation
+//
+private extension AppCoordinator {
+    func checkPendingStoreCreation() {
+        guard storeSwitcher.isPendingStoreSwitch else {
+            return
+        }
+
+        Task { @MainActor in
+            if let siteID = try? await storeSwitcher.listenToPendingStoreAndReturnSiteIDOnceReady() {
+                askConfirmationToSwitchStore(siteID: siteID)
+            }
```

Comment:
Should we do theme installation here, too? If the store is not ready yet, theme installation in the `DashboardViewModel`'s initialization will fail.

Reply:
Good idea. Done in 411a20a8b639c88f902e88a2e18a373665ef054f